src/controllers/dashboard/dashboard.js

```diff
@@ -741,21 +741,15 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
     };
     return function (view, params) {
         function onRestartRequired(evt, apiClient) {
-            if (apiClient.serverId() === serverId) {
-                renderHasPendingRestart(view, apiClient, true);
-            }
+            console.debug('onRestartRequired not implemented', evt, apiClient);
         }
 
         function onServerShuttingDown(evt, apiClient) {
-            if (apiClient.serverId() === serverId) {
-                renderHasPendingRestart(view, apiClient, true);
-            }
+            console.debug('onServerShuttingDown not implemented', evt, apiClient);
         }
 
         function onServerRestarting(evt, apiClient) {
-            if (apiClient.serverId() === serverId) {
-                renderHasPendingRestart(view, apiClient, true);
-            }
+            console.debug('onServerRestarting not implemented', evt, apiClient);
```

Comment:
why?

Reply:
`renderHasPendingRestart()` doesn't exist.

Reply:
`renderHasPendingRestart` is undefined. It used to be just an empty function, but that was removed at some point. Thought we should just log it in case this is used somewhere.